blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component, PropTypes } from 'react';
+import OrganisationPipelines from './OrganisationPipelines';
+
+class OrganizationPipelines extends Component {
+
+    render() {
+        const organization = this.context.params.organization;
+
+        return (
+            <OrganisationPipelines { ... this.props } organization={organization} />
+        );
+    }
+
+}
+
+OrganizationPipelines.contextTypes = {
+    params: PropTypes.object,
+};
+
+export default OrganizationPipelines;
```

Comment:
Another higher order component, similar to `AllPipelines` - this one passes in the organization to provide that context. Honestly we could probably just do a little extra logic in OrganisationPipelines and discard them both, but I figured I'd leave them in for now to get some feedback.